gradle.properties

```diff
@@ -1,3 +1,10 @@
+android.enableJetifier=true
+android.injected.testOnly=false
+android.useAndroidX=true
+org.gradle.caching=true
```

Comment:
I would prefer this to be off. This projects builds in a few seconds so caching doesn't really help. And Gradle doesn't have the best track record for correctness in caching.

Reply:
this could be off, I turn it on because unit tests failed just now, I tune it off and try again

Reply:
![image](https://user-images.githubusercontent.com/10363352/103012053-ec20e100-4575-11eb-9af1-669a036f3d5a.png)
@JakeWharton  It might something depends on AndroidX, should keep
```
android.enableJetifier=true
android.useAndroidX=true
```